solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -268,35 +258,90 @@ public void testAliasHandling() throws Exception {
   }
 
   @Test
-  @LogLevel("org.apache.solr.servlet.HttpSolrCall=INFO")
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
   public void testHttpCspPerf() throws Exception {
 
-    String COLLECTION = "TEST_COLLECTION";
-    CollectionAdminRequest.createCollection(COLLECTION, "conf", 2, 1)
-        .process(cluster.getSolrClient());
-    cluster.waitForActiveCollection(COLLECTION, 2, 2);
+    performTest(false, "HTTPCSPTEST",
+            1,
+            10,
+            //1 create collection, 2 /admin/core , 1 LISTALIASES, 1 CLUSTERSTATUS, 10 CLUSTERSTATUS with coll param
+            15,
+            15);
 
-    SolrInputDocument doc = new SolrInputDocument("id", "1", "title_s", "my doc");
-    httpBasedCloudSolrClient.add(COLLECTION, doc);
-    httpBasedCloudSolrClient.commit(COLLECTION);
+  }
+  @Test
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
+  public void testZkCspPerf() throws Exception {
 
-    for (int i = 0; i < 3; i++) {
-      assertEquals(
-          1,
-          httpBasedCloudSolrClient
-              .query(COLLECTION, params("q", "*:*"))
-              .getResults()
-              .getNumFound());
+    performTest(true, "ZKCSPTEST",
+            0,
+            0,
+            //1 create collection, 2 /admin/core
+            3,
+            15);
+
+  }
+
+  private CloudSolrClient createHttpCSPBasedCloudSolrClient() {
```

Comment:
"Csp" capitalization.
Why put "based" in it... The corresponding ZK one does not have that.  I suggest naming them both in harmony like createHttpCspCloudSolrClient and createZkCspCloudSolrClient and which are both plenty long names as it is

Reply:
Removed the zk test